bridge/contracts/test/nftbridge/TestTokenCreator.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.6;
+
+contract TestTokenCreator {
```

Comment:
couldn't we just use the NFTERC721TestToken contract for this?

Reply:
We could, but which address should be returned? The creator() function doesn't have tokenId as a parameter 
Do you think we could use the `_msgSender` as the returned value?

Reply:
Actually, we need another contract because we want to test the other edge from the `getTokenCreator` function